lib/Controller/Display.php

```diff
@@ -573,6 +581,12 @@ function grid(Request $request, Response $response)
             $displayProfiles[$displayProfile->displayProfileId] = $displayProfile->name;
         }
 
+        // Get all Display Types
+        $displayTypes = [];
```

Comment:
Instead of having this separate, we can have a `LEFT OUTER JOIN` in the displayFactory which sets the display type from there.